less/storybook.less

```diff
@@ -0,0 +1,3 @@
+.crane-option-selected {
+  color: rgb(94, 158, 214);
+}
```

Comment:
@vholcomb78 and @sthomas1618  - Let me know what you think about this.

Reply:
Looks good to me

Reply:
LGTM